Sources/Rownd/Views/HubWebView/HubWebViewController.swift

```diff
@@ -232,10 +232,12 @@ extension HubWebViewController: WKScriptMessageHandler, WKNavigationDelegate {
                 guard case .authentication(let authMessage) = hubMessage.payload else { return }
                 guard hubViewController?.targetPage == .signIn  else { return }
                 DispatchQueue.main.async {
+                    if (store.state.auth.accessToken != authMessage.accessToken) {
```

Comment:
Won't this always be true?

Reply:
Yup you're right. Not sure what I was thinking with this one. I'm gonna close the PR